isis/src/base/objs/ProcessGroundPolygons/ProcessGroundPolygons.cpp

```diff
@@ -11,7 +11,7 @@ find files of those names at the top level of this repository. **/
 #include "geos/geom/CoordinateSequence.h"
 #include "geos/geom/CoordinateArraySequence.h"
 #include "geos/geom/LineString.h"
-#include "geos/geosAlgorithm.h"
+//#include "geos/geosAlgorithm.h"
```

Comment:
Can this line be removed?

Reply:
Stuart explained a rationale for leaving removed includes as comments in a call earlier that sounded reasonable to me. Something about being non-destructive while we make these rapid changes